### PR TITLE
feat: adds Logo component

### PR DESCRIPTION
**Pull request by OlegSymonchuk**

### CODE REVIEW

**Review by kucherenko — APPROVED** [review]

File: components/common/Logo/Logo.interface.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export interface LogoProps {
```
Comment (kucherenko) [inline components/common/Logo/Logo.interface.tsx]: ts not tsx

File: components/common/Logo/Logo.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+import LogoImgBottom from './LogoImgBottom';
+import LogoImgTop from './LogoImgTop';
+import { LogoProps } from './Logo.interface';
+
+const availableColours: readonly string[] = [
+  '#0687FF',
```
Comment (kucherenko) [inline components/common/Logo/Logo.tsx]: colors should be in single global place

**Review by kucherenko — CHANGES_REQUESTED** [review]

File: components/common/Logo/Logo.interface.tsx (hunk 1 above)
Comment (OlegSymonchuk) [reply components/common/Logo/Logo.interface.tsx]: Done

File: components/common/Logo/Logo.tsx (hunk 2 above)
Comment (OlegSymonchuk) [reply components/common/Logo/Logo.tsx]: Moved

**Review by kucherenko — APPROVED** [review]

**Pull request merged**